inc/common.class.php

```diff
@@ -247,12 +247,12 @@ public static function cancelMyTicket(int $id) : bool {
     *                +-------------+---------+---------+----------+
     *                |NULL or NONE | WAITING | REFUSED | ACCEPTED |
     *     + ---------+-------------+---------+---------+----------+
-    * T S | INCOMING |     T            V          V         T
-    * i t | ASSIGNED |     T            V          V         T
-    * c a | PLANNED  |     T            V          V         T
-    * k t | WAITING  |     T            V          V         T
-    * e u | SOLVED   |     T            V          T         T
-    * t s | CLOSED   |     T            V          T         T
+    * T S | INCOMING |      T           V         V          T
+    * i t | ASSIGNED |      T           V         V          T
+    * c a | PLANNED  |      T           V         V          T
+    * k t | WAITING  |      T           V         V          T
+    * e u | SOLVED   |      T           T         T          T
+    * t s | CLOSED   |      T           T         T          T
```

Comment:
recentered values in matrix, and changed value for solved or closed ticket and global_validation is waiting